ping.go

```diff
@@ -122,6 +123,9 @@ type Pinger struct {
 	// Number of packets received
 	PacketsRecv int
 
+	// Number of packets received duplicates
+	PacketsRecvDup int
```

Comment:
This should probably be a `uint64` so that the type doesn't change between platforms. Also, we don't want negative values.